Formula/qbs.rb

```diff
@@ -1,8 +1,8 @@
 class Qbs < Formula
-  desc "Qt Build Suite"
-  homepage "https://wiki.qt.io/Qt_Build_Suite"
-  url "https://download.qt.io/official_releases/qbs/1.5.0/qbs-src-1.5.0.tar.gz"
-  sha256 "541106d3e53429c5375a58f395413b3cd5a026d91a304a10f36d78b5e39d9085"
+  desc "Qbs"
```

Comment:
Can you please provide a more descriptive description? Replicating the formula name isn't very useful to someone not familiar with the project and doesn't even offer a vague idea what it is about. The previous “Qt Build Suite” was fine, but maybe that's no longer fitting and maybe it can be made even better.